go.mod

```diff
@@ -13,7 +13,7 @@ require (
 	github.com/oxtoacart/bpool v0.0.0-20190530202638-03653db5a59c
 	github.com/sirupsen/logrus v1.9.0
 	github.com/stretchr/testify v1.8.0
-	go.k6.io/k6 v0.42.1-0.20230109152238-8418147fa630
+	go.k6.io/k6 v0.42.1-0.20230130080633-582ec4d3940c
```

Comment:
Note: I've updated to the latest `master` version of `k6`.

Reply:
Let's do that in another PR and let's merge to `main`. Then this will get it from the `main`. We don't something to fail if we add something interim before the release.